### PR TITLE
Added TPLOCs for Binance trades/deposits/withdrawals

### PR DESCRIPTION
**Pull request by anson-vandoren**
Added TPLOCs to allow for importing of csv files from Binance for trades, deposits, and withdrawals. Fully tested, and ready for review.

This is my first time submitting a pull request, so please let me know how I can improve.

### CODE REVIEW

File: ccgains/trades.py [hunk 1]
```diff
@@ -560,17 +560,19 @@ def append_binance_csv(
             self, file_name, which_data='trades', delimiter=',',
             skiprows=1,  default_timezone=tz.tzutc()):
```
Comment (probstj) [inline ccgains/trades.py]: Everything good so far, but could you add a docstring here please? Just copy & adapt the one from append_poloniex_csv, then you'd merely have to add 'distribution' to the explanation of `which_data`.

File: ccgains/trades.py [hunk 2]
```diff
@@ -549,6 +556,28 @@ def append_ccgains_csv(
             skiprows=skiprows,
             default_timezone=default_timezone)
 
+    def append_binance_csv(
+            self, file_name, which_data='trades', delimiter=',',
+            skiprows=1,  default_timezone=tz.tzutc()):
```
Comment (probstj) [inline ccgains/trades.py]: Everything good so far, but could you add a docstring here please? Just copy & adapt the one from append_poloniex_csv, then you'd merely have to add 'distribution' to the explanation of `which_data`.

**Comment by probstj** [comment]
Thank you very much for your additions! 

I am just wondering about the implications of adding distributions / airdrops, especially regarding how they must be treated for tax calculations. I think it would make most sense if they are added as new bags with base cost of zero, right? Because you paid nothing for them, then those coins must count 100% towards capital gains (but only in case you eventually sell them before holding them more than 1 year).

File: ccgains/trades.py (hunk 2 above)
Comment (anson-vandoren) [reply ccgains/trades.py]: Done, and thanks for the constructive feedback!

**Pull request merged**